src/style.css

```diff
@@ -0,0 +1,109 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+.searchInput {
+  border: none;
+  outline: none;
+  position: absolute;
+}
+
+.searchBox1 {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 68px;
+  margin-top: 20px;
+}
+
+.searchInput1 {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-top: 23px;
+  padding-bottom: 20px;
+  padding-left: 18px;
+  font-weight: 300;
+  font-family: inherit;
+}
+
+.searchImg1 {
+
+  width: 19px;
+  height: 19px;
+  padding-top: 24px;
+  padding-left: 25px;
+}
+
+.searchInput2 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  padding-top: 10px;
+  padding-bottom: 8px;
+  padding-left: 10px;
+  font-family: inherit;
+}
```

Comment:
move all repetitve rules from `searchInput1` and `searchInput2` to the common `searchInput` class. Use same approach with searchBox and images